src/rebar_state.erl

```diff
@@ -289,7 +289,13 @@ apply_profiles(State, Profile) when not is_list(Profile) ->
 apply_profiles(State, [default]) ->
     State;
 apply_profiles(State=#state_t{default = Defaults, current_profiles=CurrentProfiles}, Profiles) ->
-    AppliedProfiles = deduplicate(CurrentProfiles ++ Profiles),
+    AppliedProfiles = case Profiles of
+                          [global | _] ->
```

Comment:
Should we have a comment about something like "global mode is a special profile for plugins" or something like that? If I recall it does not clash with a standard `default+global` profile set because `default` is always first, *except* when installing global plugins?